docs/api/doc.md

```diff
@@ -0,0 +1,470 @@
+---
+title: Doc
+layout: default
+parent: API
+
+copy:
+  event_source: This will be `false` for remote ops received from other clients, or will be truthy for ops submitted from this doc instance. For local ops, it will be the value of `source` supplied to [`submitOp`](#submitop), or `true` if no value was supplied
+  submit_source: An argument that will be passed to local event handlers for distinguishing how different ops were produced. This will only be sent to the server if [`submitSource`](#submitsource--boolean) is set to `true`
+---
+
+# Doc
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+The `Doc` class is the client-side representation of a ShareDB document.
+
+A `Doc` instance can be obtained with [`connection.getDoc()`]({% link api/connection.md %}#getdoc).
+
+## Properties
+
+### `type` -- [Type]({% link types/index.md %})
+
+> The [type]({% link types/index.md %}) of the document
+
+{: .info }
+> If a document has been fetched, and its `type` remains unset, then the document has not yet been created.
+
+### `collection` -- string
+
+> The document's collection
+
+### `id` -- string
+
+> The unique document ID
+
+### `version` -- number
+
+> The latest version of the document fetched from the server. As ops are received from the server, `version` will be incremented.
+
+{: .info }
+> The `version` will only be incremented for local ops sent through [`submitOp()`](#submitop) after the server has acknowledged the op, when the `submitOp` callback has been called.
+
+### `data` -- Object
+
+> The document contents
+
+{: .warn }
+> The data will only be available after a document has been fetched or subscribed to.
+
+### `preventCompose` -- boolean
+
+> Default: `false`
+
+> Set to `true` to prevent ops from being composed together. This is read at the time of calling [`submitOp()`](#submitop), so it may be toggled on before submitting a specific op, and toggled off again afterwards
```

Comment:
Have we explained elsewhere what it means if ops are composed together?

Reply:
No, but the original `README` doesn't mention it either. I'll add a TODO next to this to add some documentation on composing.